hello-microservices-middle/src/main/java/com/matera/hellomicroservices/entities/Address.java

```diff
@@ -0,0 +1,96 @@
+package com.matera.hellomicroservices.entities;
+
+public class Address {
```

Comment:
refactor this to use Builder pattern. 

usage example:

`Address addr = Address.builder()
.withCity(...)
.withState(...)
.withCountry(...)
.withZipcode(...)
.build()`

the build method should validate the fields value and format (if needed), set all attributes as final and block direct instantiation, force construction through its builder.